manifests/init.pp

```diff
@@ -27,6 +41,8 @@
   Boolean                    $manage_repo              = true,
   String                     $package_ensure           = installed,
   String                     $package_name             = 'gitlab-runner',
+  String                     $repo_base_url            = 'https://packages.gitlab.com',
+  String                     $repo_keyserver           = 'keys.gnupg.net',
```

Comment:
you can probably use this datatype https://github.com/puppetlabs/puppetlabs-stdlib/blob/master/types/fqdn.pp

Reply:
Yes, done for keyserver, Thanks.